tests/Agent/IntegrationTests/IntegrationTests/BasicInstrumentation/MsCorLibTests.cs

```diff
@@ -37,7 +36,7 @@ public MsCorLibTests(RemoteServiceFixtures.BasicMvcApplicationTestFixture fixtur
                     {
                         _fixture.Get();
                     }
-                    catch (WebException ex)
+                    catch (AggregateException ex)
```

Comment:
Do we want to pull out the individual exceptions in this?

Reply:
I don't think it really matters here -- the top-level exception that's getting thrown from `HttpClient` is `AggregateException` - which is as close to "equivalent" as I can get to `WebException` from `WebClient`.

I think this is a case of optimistic coding for a case that never happens -- at least, I've never seen this test fail with the exception that gets thrown on line 41.